microsoft-edge/webview2/how-to/winrt-from-js.md

```diff
@@ -343,8 +353,34 @@ Next, use the DevTools Console to demonstrate that web-side code can call the in
    ![Using the DevTools Console to test calling native-side code from web-side code.](winrt-from-js-images/devtools-console-calling-native-side-code.png)
 
 Congratulations!  You've finished the sample demonstration of calling WinRT code from JavaScript code.
+## Custom (3rd Party) WinRT Components
+
+The WebView2 WinRT tool supports custom WinRT component projections as long as its using a C# base WinRT class.
+
```

Comment:
@mikehoffms can we embed this picture somewhere here: <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAdEAAAEvCAYAAAAaW/vyAAAgAElEQVR4Ae2d+48cxdnv8z9NIGDuiBCuwkACgh9sEMF4FWK/3E4ghBPQCcuagBwgnACyEZyY5RqzxBB8DgHsrBK9BgIxCMl+DYSLwERaIwIv2JBLHz3VXdVV1dUzPbN9n89Kq5npqa56nm899Xy6qi/zjYg/FEABFEABFECBiRT4xkR7sRMKoAAKoAAKoEAERAkCFEABFEABFJhQASA6oXDshgIogAIogAJAlBhAARRAARRAgQkVAKITCsduKIACKIACKABEiQEUQAEUQAEUmFABIDqhcOyGAiiAAiiAAkCUGEABFEABFECBCRUAohMKx24ogAIogAIoAESJARRAARRAARSYUAEgOqFw7IYCKIACKIACQJQYQAEUQAEUQIEJFQCiEwrHbiiAAiiAAigARIkBFEABFEABFJhQASA6oXDshgIogAIogAJAlBhAARRAARRAgQkVAKITCsduKIACKIACKABEiQEUQAEUQAEUmFABIDqhcOyGAiiAAiiAAkCUGEABFEABFECBCRUAohMKx24ogAIogAIoAESJARRAARRAARSYUAEgOqFw7IYCKIACKIACQJQYQAEUQAEUQIEJFQCiEwrHbiiAAiiAAigARIkBFEABFEABFJhQASA6oXDshgIogAIogAJAlBhAARRAARRAgQkVAKITCsduKIACKIACKABEiYFaFNj17IGI/25rUEug0AgKdEwBINqxDuuquf/57IHoxa38d1WDu/5jX1dDD7tRoFIFgGil8lK5VkBDdGkpivjvngZAVEcyryjgKgBEXT34VJECQLR74LQPdoBoRQODajuvABDtfBd2wwEgCkS7EalYiQLjKQBEx9OL0hMqAESB6IShw24o0GoFgGiru6c/xgFRINqfaMYTFEgVAKKpFryrUAEgCkQrDC+qRoHGFACijUk/XQ0DUSA6XRGPt9OiABCdlp5u2E8gCkQbDkGaR4FKFACilchKpb4CQBSI+jHBZxTogwJAtA+92AEfgCgQ7UCYYiIKjK0AEB1bMnaYRIE6ILpvfl00GAzc/7UL0b7SnpK0O5odDKKZ+aVWPHVJ/K3LFh62MEnUs880KABEp6GXW+BjfRBdF23Zo2d9S9GWtQJVe5v+bpLXaiAaw39jtDgG7PUBAxBtQXBjwlQrAESnuvvrc74ZiEbR0p6FaKZFs0f7UXr6/bgQXZxLZ9tAtL4YpiUUCCkAREOqsK10BRqD6FI8exzM7Y40rLZYy74GQglszXKwtwysZ37q+7ndznKu850sJ3vfCyxt8Ekdut3MvoNBNLsjf5Ys5dX3NR8csJxb+pCgwp4oAER70pFtd6MxiFqwMcDyALm0Y6M6j6rBtrSkl4H9JdZkuwXJGI52uQTapo1QXVImXWKO7bLryIeonr3WPcMGom0fYdjXlAJAtCnlp6zdZiDqAiyGVQqvGEhJGQO9BGAJfN1ZoQdRVcavL4piKCfbFaADZazzn0B0ygYD7vZKASDaq+5srzP1QTQ9X6iWXi04hmGVd7FQaLsL0bg+rz1zdXAMzuxMNTvLDNuVLWdmoQJga4btbLfgXOZ2ZqLtHVtY1qwCQLRZ/aem9fogmj/rC8MqBEsBWGh7CKL57QnEFEQtkIfAFrYLiE7N4MDRTisARDvdfd0xvr0QXcZybnIu1V3ydeEXA3I4aIFod+IYS1HAVwCI+orwuRIF2gtRfQ7Tvio2B6z6gqPMhUUuJBfn7M/xjHYwsC8ckvqtzwVgnJnBspxbSZxSKQqMqwAQHVcxyk+kQKshKucRE5Dl3eISQ8xdztVg829fGQggzQMfZGaa7GfOl8otLrujfVYZp44Ry7+qXSA6URyyEwqUrQAQLVtR6gsqUAdENdSqew1DtLr23KXhJtvhwqJgWLMRBSIgShDUokAvIDrJsmtFV8vWDVQgWsswoZEOKgBEO9hpXTS5yxCNL/wp8xm87ZlhFoUxEO3iqMPmOhQAonWoTBtRlyFaFDR9LgdEGcQoEFYAiIZ1YWvJCgDR7s0+7YMCIFrygKC63igARHvTle12BIgC0XZHKNahwGQKANHJdGOvMRUAokB0zJChOAp0QgEg2olu6r6RQLQBiCb3kg57opK9ZDvsPcu53R+DeFCNAkC0Gl2p1VNgXIgWefiAXDVrfr5MgFHkIQVDbjmR+soAzjAY5X/nPZDBeipS/j4jwAxEvSjkIwqUrwAQLV9TagwoUBiioSfxyP2ZGUAKdNLH6zlAHQLKfCAlj+fLtDMCVBO15dUZuP9U3VazXJAC0UAksgkFylUAiJarJ7XlKFAMoqFfTvGAY6C1O5q1gOc+rzZvnyHbBWRzG50fy84H7pB6jH0Fy5QIuoy9JdbNcm5OYLN56hUAolMfAvUIUASiavblPKg9BCJv2dN6Hq0899Ys744Fs7hOWcqVZWS/Dm3X7Jz926HpLFjgVaRMBnIFfyot3s/3220/WGbtumhmYD9YP/0dUvOM4JF6x30AROsZJ7TSPQWAaPf6rJMWj4bof0db1g4Cy7YhkMawM+cvZca1nKVPNWNLflVFLa1av7BiAOnCKIZmus3/nII1LZOFaLKEPNL27FJztr1sGf1QfaNTYNk4PvccArKrOxDt5LDD6BoUAKI1iEwTUYEnFo0DUZmVpaAToBhQjDUDjUGhQGJAFsPIri8GVtqeM+tL9itSZlKIxnX7oEtmpsPad5Zz4/L+LDv84+MuQMVuIMooRoGwAkA0rAtbS1ag1JmowMGcDxU4+IDJQiALMF0mC00FVVN/ulS76AHaLheGaDxjzl4U5bY9MADX291Xux3bj3S7C1RTxoZo8j5dxrWXprNL2KaOxGcgWvKAoLreKABEe9OV7XZkNEQT4Iw4R6fA4Z0H1WCwZ48+BPI+x/BzgRLXl4K5CCCLlMnakMDPAna2TD6IJ4FodibqAjvUvmwDou0eX1jXnAJAtDntp6rlIhDV5/BGJXqBhwHmjo2ZC4HyQJDdnpxHzMwE4+3ajjAg3X2LlMm2r2e5lj/ebFf2ietOoR7X484+U6BaULRnokuuvSFbhm0DolM1XHF2DAWA6BhiUXRyBQpBVF+t6l9lq2Cgz0kKDPR77wIjC0DxjNUHjwUYKRu40EaDJN4/bicEMb/+ImV03e5rAkP/KlqxzcxQswDMtJf44iwN+/4ln/XBgbKj4EEIEJ089tmz3woA0X73b2u8KwpRndj1Eq1+nZlfiBZ3JOAzM0cXqDacYsgNO9en4ZUC2d7fBqwG1oxcPWyWkl1AFynj1G8BX7bH+9v1D6LZ+d3R4h4N/gSkOe2ruv3znqFbXDRsrXpmdyxFw2yT74Boa4YShrRMASDasg7pqzljQdQDzKgEH/4+hqRZ9l1GnTHgcmCb1FukTNhODcl2vwLRvo5M/FquAkB0uQqyfyEFaoeozLjMcujyAFUEkEXKANFCoUIhFOiUAkC0U93VXWNrhWiJABXwFQFkkTJAtLvxi+UokKcAEM1Thu2lKlArRJexdNtl0FVpO8u5pQ4HKuuRAkC0R53ZZleA6PKWlKsEZJG6gWibRxe2NakAEG1S/SlqG4gC0SkKd1ydIgWA6BR1dpOuAlEg2mT80TYKVKUAEK1KWep1FACiQNQJCD6gQE8UAKI96ci2uwFEgWjbYxT7UGASBYDoJKqxz9gKAFEgOnbQsAMKdEABINqBTuqDiUAUiPYhjvEBBXwFgKivCJ8rUUBD9MWtByL+u6cBt7hUMiyotAcKANEedGIXXNj17IFo2P/8L1+K/sf3740uOfuW6NrLNg0tO6wevhuus63P/7njT0pv0XzDtb+JFp/aP1T3LsQZNqJA3QoA0boVpz1HgVdffTW68sorza+j3HjjjdGePXucMnyoRoHPPvssuvfee432xx9/vPpcTWvUigL9VACI9rNfW+/VBx98EG3YsMEk8NWrV0fbt29vvd19NPDDDz90DmTOPPPM6JVXXumjq/iEAqUrAERLl5QKRynw0EMPRSeddJIC6LHHHqtmPwcPHhy1G99XrICA84wzzjAHNldddVUkgOUPBVAgXwEgmq8N35SswBtvvBGtX7/eJOkf//jHLN2WrHEZ1ckS73HHHaf6iSXeMhSljj4rAET73Lst8m3z5s3RkUceqRLzOeecEz3zzDMtsg5TfAXkfOlPf/pTc8DDEq+vEJ9RIFYAiBIJlSrw8ssvR5dddplJxnIedEl+NoS/TiggS7wXXHCB6T+WeDvRbRhZowJAtEaxp6mpr776Krr77rtN8l21alW0c+fOaZKgV77KeWx7iXd+fr5X/uEMCkyqABCdVDn2y1VgcXExkqttB4OB+heYHjp0KLc8X3RDAVnitW9Hkhnq3r17u2E8VqJARQoA0YqEncZqv/zyy+j222838JyZmeFWiR4Ggn8Vr/S5AJY/FJhGBYDoNPZ6BT7v3r07uvjiixVA5baVBx98sIJWqLItCgg0b7vtNnPAxIVHbekZ7KhbASBat+I9bO+JJ56Ijj76aJVQL7nkEm5b6WEf57kky7n2hUfMSvOUYntfFQCife3ZmvyanZ01sxG571MuKOJv+hSwHx8os1LOlU5fDEyrx0B0Wnu+BL/18q1cQHTHHXeUUCNVdFkBf1YqYOUPBfquABDtew9X4J88IF4v3wpAn3zyyQpaocquKmCfK12zZg2PDuxqR2J3IQWAaCGZKKQV2Lp1q1m+/c53vhPJg+T5QwFfAbmC176v9MUXX/SL8BkFeqEAEO1FN9bjhFxxq+/9vPTSS+tplFY6q4B/XynLu53tSgwfogAQHSIOX6UKPPXUUwagJMNUF96NVkCedqQPvmR5l3tKR2tGie4oAES701eNWfrcc8+ZJMj9n411Q6cblouO9PIuV+92uisx3lMAiHqC8NFVwAboo48+6n7JJxQYQwGZgep7SuUn1jhPOoZ4FG2tAkC0tV3TvGE2QB9//PHmDcKCzisgILV/Yu23v/1t533CgelWAIhOd//nem8DVM5p8YcCZSpg3wYDSMtUlrrqVgCI1q14B9qTZTZ9IcgDDzzQAYsxsYsKCDx1nAHSLvYgNosCQJQ4cBT45JNPovPOO08lt/vuu8/5jg8oULYCgLRsRamvbgWAaN2Kt7y9m2++WQH06quvbrmlmNcXBWyQykMa+EOBLikARLvUWxXbum3bNgXQk046KXrrrbcqbo3qUSBVQD/AXm5/4T7SVBfetV8BINr+PqrFwvfffz867bTTFES5ErcWyWnEU0CegiXnSK+66irvGz6iQHsVAKLt7ZtaLbvuuutUArv++utrbZfGUEAr8OGHH5oHMnChkVaF17YrAETb3kM12PfII48ogMpMVGak/KFAUwro86PyMAaBKn8o0HYFgGjbe6hi+w4cOBCdcsopCqLyfFz+UKBpBa688koVj/JQBv5QoO0KANG291DF9t1///0qYV1++eUVt0T1KFBMAZmB6vtHmY0W04xSzSkARJvTvvGWP//88+iss85SCWv79u2N24MBKKAV0I8G5BeDtCK8tlUBINrWnqnBLv0TVatWraqhNZpAgeIKyP2iMhuVc6Pc8lJcN0rWrwAQrV/zVrT49ddfmycTPfbYY62wCSNQwFZA3/LClbq2KrxvmwJAtG09UpM9ci+oHOmvXLky+vLLL2tqlWZQoLgC+kpdgSl/KNBWBYBoW3umYrsuuugiBdFNmzZV3BLVo8BkCsgyrr7AaLIa2AsFqlcAiFavceta+OCDD0xyeuedd1pnHwahgFZA/4g3z9TVivDaNgWAaNt6pAZ7FhYWFEQvvPDCGlqjCRSYXAH9u6NcpTu5huxZrQJAtFp9W1m7vn3g1ltvbaV9GIUCWgH927acF9WK8No2BYBo23qkBnvklzLkXNPzzz9fQ2s0gQLFFVixYkV0xBFHRMccc4z6P/roo82pB71Nv0q5p59+unjllESBChQAohWI2uYq5SfOBKCHH354dOjQoTabim1TqIAs29rg1BcWhV4FovyhQNMKANGme6Dm9vXD5tesWVNzyzSHAqMVkCtyjzrqKDP7DMFTb5Ny/KFA0woA0aZ7oOb25bcaJQk98MADNbdMcyhQTIEis1GZhXKxUTE9KVWtAkC0Wn1bV/vpp5+uIPrmm2+2zjYMQgFRoMhsVGahPA6QeGmDAkC0Db1Qkw3ywHm9FCaP/eMPBdqqgMwyZbap49V+ZRba1l6bTruA6BT1+2uvvaaS0rnnnjtFXuNqFxUYNhtlFtrFHu2vzUC0v32b8Uw/L/e6667LfMcGFGibAqFzo8xC29ZL2ANEpygGbrnlFjUTlR/i5g8F2q5AaDbKLLTtvTZ99gHRKerzSy65REF0cXFxirzG1S4rILPRww47TMWtvHJFbpd7s5+2A9F+9mvQq+OOO04lo48//jj4fV0bdz17IOIfDYrEwOJT+6M1390Qff+cW9SrfC6yH2WIL4mBOv6AaB0qt6CN/fv3K4CecMIJjVsjwf3iVv7RgBggBqqLASDaeKrvlwG7du1SEG3Dk4o0RJeWooh/NCAGiIGyY0AOToBovxjWuDcvvPCCgugVV1zRuC1AlKRZdtKkPmLKjgEg2nia758B27ZtUxC94YYbGncOiJLw7ITHe+Kh7BgAoo2n+f4Z8PDDDyuIzs3NNe4cECVplp00qY+YsmMAiDae5vtnwKZNmxRE77rrrsadA6IkPDvh8Z54KDsGgGjjab5/Btx5550Kops3b27cOSBK0iw7aVIfMWXHABBtPM33z4ANGzYoiM7PzzfuHBAl4dkJj/fEQ9kxAEQbT/P9M+Cmm25SEH3yyScbdw6IkjTLTprUR0zZMQBEG0/z/TPg2muvVRDdvn17484BURKenfB4TzyUHQNAtPE03z8D1q9fryC6c+fOxp0DoiTNspMm9RFTdgwA0cbTfP8MkCcVyQ8bv/TSS407B0S7mPCWoi1rByqG7B/InplfKu+pUzs2RoPBumjLnqb0yfpYqn88oau8WBmhJRBtPM33z4DVq1erBPj666837hwQbQoSy2k3AczahWifTmB7FqKZwSAa2Nv0d5O8VgLRxO653SMT+L75da4viX+zO5ajG/vaM8S63gPRxtN8/ww4//zzFUT37t3buHNAtIuJNQDRpShS4Gl09jhKy+IQzSb43dGsHCQUAHB231F28X2VmgHRxtN8/wxYuXKlguh7773XuHNAtIsJNAzRJTV7HESzOzSsFqxlX2tpNilnloIdMCX7CrAGUperz+Kcu4ycXWJNYJfsny4Ju/XGbW+MFgvPkoFolaCrsm4g2nia758Bp556qkpQTf+WqCgLRF1IVJlMyqs7DNF0JpoCy4dcXMaGYwInfxk4s3waaFMvIWsI+58FkAJsU3dShy5fGKB6lm3b3cV+m06bgWj/GNa4RyeeeKKC6Kefftq4LUC0i4ltFNAC3ytg5czmQuc/fYiqMtmZYwzleLuapRpghnSdEKIazkPrDrXHtvIO3CbXEog2nub7Z8CKFSsURA8ePNi4c0B08uTQXIJKYGSWTOMl1nTWmQMrH4x6Jhja7m3zl3HNUrCyQSCaA2jdhnrNscsp4/eH9tVajh5a3t+fz83Faaw9EG08zffPAJ2A2uAZEO1ikk3Akjszy4GVB0aTXEPbvW0xRLMzUVNHAtEU5CFdc+waAkW9/Dy83lBbbEv7plktgGgbMn3PbACizQ7qtiSXye2YEKJ5s8UCy7np+da8vhtlk+w3JkQTkKfnVPPaZvvksVS9dkC0ZwBrgztAtPqB2+aksnzbRgErH1ZZGOYsw3ozUQPAgT0blXaszwrG3m0oUo91IdHoGa2OjcSHwBXCy9dPt8FrHVoC0TZQp2c2AFGS1/KS1+QQlXb1EqmOw+C9lxmISp+lYEv3XYj22U810rNHc752XbRlx1L6UAg9G06+z1umzdho6uMK3eXFTv1jD4j2DGBtcEcnoDbYwjnR+pNKJ5JgEKJo1Ym+G3KOuQn7gWgbMn3PbACiJOMmktk4bRZfdqUvx9F1GssC0Z4BrA3uAFESbzuTqbVcm3vlL33Xzr5rb78A0TZQp2c2ANH2DngSJH1DDJQbA0C0ZwBrgztAtNxBStJDT2KgvTEARNtAnZ7ZAETbO+BJxvQNMVBuDADRngGsDe4A0XIHKUkPPYmB9sYAEG0DdXpmAxBt74AnGdM3rYqBxm41Si4ysx6UMakuQLRnAGuDO12GqPMg8hqu4IxvureeijP2PXA5T+QZu56WwiX0cAP74QcT+Jl9qlFLfZ/At0lB0Nh+QHSslP2NsUpTuLMKdBKiyWB2njAjj3mrFKTx0bDT5tiJs7wjakmky4d6mUDK6lOKfaFn6Y6tu/azXP0bg1lh/0v2F4iOleeB6Fhydbdw9yAaz+aWBzOdVMd4LTWZj9HukIRZCqSG1L9sSCTPr53dUY6/y7ZHPyqwhGXB5dtShyZA1O8nlnO7y6rWWt41iI4NjpEz1DjRjEr0aunYTr4hqMqRul3GmS0mS7ny3FVdRh3Zy29TWt8NBpFzgKDbmd+ofvdV9dfc7uwzZ8d+OHo4wS7O+b+VqQ9a9kVb1g6ys/3QqoAGswdRpaG3WiD96Wqf2hX3tfw+aWqT2iZ1JHXH8Zt+L0lTx8iW+XVGs5n5xH7rubcD5wH2Wail7ce/kaq1TxOz22+2nVJGx4xbT/Z0gPu9FR9KR63/QjRrbE/qSLTXY9j1J9HR7CM+uG07p0P8uFNte3WsXRfNFImzoXZF0ZLTd76/cT+4mqyLZiT29LhJ4mu0/dk+BaKtRVF3DdMDsA0ejH527hfhRK6TduhVBqyXuNMkKIMsThRuIvcHX5zI3DI6uS1Fur544NsJPQVCXCZJujoZmGRjJTcPPGnCscokfsbtZbdre0a9Zvf37JN2lD2JT/b7xIYQGON2dQK27Mvsn5Sx+yfRRGvtaxp/dhNqnEzTdkwZu15lr98ffj+nnzPa6H6ZTx5gn9hpH/DodrXtOsmnZbL++rYv6YfiG9uTPsnAS+oKxJrZL41tHz465p1xoWNRx2bGDh0LIx66n+iU+pwc1CT1+hpl/U0OPqwDJ1PG2JbVcSljf9qX9jgAom3I9D2zYcWKFepo/eDBg4171lqIqsSQJmk9KF2ASLKLj5hNAvGAkEkGme9l4MdJ09Shk3dgSTST6BOwaftGvnrtq/rUbCP1NeujPVP2bLXa18nS+KG+88qLb6q9FAa+T/Hn/O+Vj0ojv0z6OdWhKERD5WLbNZCULiap64Tt7udqF5dx/FP6B+x0/HHbTX3RbVqvzn6y3bXH7KvKpX2st9u22e/19xpU+iDBbDf9ntOe+T7HF9tuLybjNrx6R9j/qmnP0ibZBkQbT/P9M+DEE09UEP30008bd648iCaD1VnKipfkdFKPk4S1TGfK+sklHsB6Pydx2ElQBrbMAtS2uA7VhjMz8JJIMGF4ZewE4yWHYKLzyjj2Br5LYRD7KQlStsX+xrbYSdNpMyeZ6WTrzHSStm1NdDtqmwJSVuu4vRQ0TvvaH2VHOkMKllFlvWSs98+8hsrZ/ZK8NzHjxVEC13yIxv7Ednr7mjq1z3FbwfjTkDT7SF16v3yIKrucfWwbJHZD/keR7lc7Hpz4SuI5bOuQ/a1xEO47155R9gPRxlP5dBlw6qmnKoh+/PHHjTs+GqJ6qccHXfaI0wxuDbdMotT7xAN0eGKwE5PeL01SM/O71TJzXIeuL351E4qdiPOSilemYojGy7Ubo0VbJ/1ete1rHds3uyPkX6qJm8wtzVTClN/1XIhm9Dk6tW1jtKi/s26LiZNqqn8wySo7y4Roel51UceN04bXR7qM91oMoqlvJmadeuK23DgSPRMb7IM0ZaNdX9xHevas648h5Per1UfLhOjwsZT2k7bHhnOwfz17Rttv++K+ZybaeJrvnwErV65UEH3vvfcad64IRPU5wmxScQeLGaCSWOxE4yQo2UdDL7y/GtSZZbu0bDzovQtukoSbBYmXfK0jcGOvTo66zUxi9NselgzTsmn9/rbEJuecW5J8Zbai7bB0Mz5rCAa+y++ftG67jJldeH0Vt5WCIf7s+ZzorZN3sIyyMQyVsDapnfq6AV2/jpnhcZUc8A3zx7M7bEcORANxEWuYamXs9PrQ1zTUrqrLs92GXWgfA3avvbSsF/86bmxf7Pf6ew+iRexP23TjHYg2nub7Z8D555+vILp3797GnSsEUX3Vo381oZ7NmIGXDB4ZlH4ycMrEyTJNkPagiwd9+LukXAJCGwh5ySuTZJYJUX1AkbEvqTcEwFByCR7ZB5OZ63Omft1uAK52u8EkGNRCzwhTMAQBqWxNZzjBMkmfB3114qFg3GhfbWDINutzCES+77E9qX+ik3uFdA5Efb12bIxm5jZmrp4N+6sPEOyDEdlmfU40dfrY09nuU/0+9k+fDki11OPD9z8zJvRBpBNDPnwL2B/q06UoAqKNp/n+GbB69WoF0ddff71x54pCVA1YPcitczsz8wvRYuACHD3Ax35VbViJJWdgLs5ly0iyyMBNJwidaP1EqOr3EoaywU2yth9xkkzOaemDBZ3gnUSUJLSQD3sWotn59CrjuP7d0ayuL7CPm+ilbp3Y7PNr+r2vz+5oVmtg6vaSeLLdT7rxZ68+L7kHy5h2En2TuNHJ3dZUvQ/El8xInT41Oms/B0rHfUlbRSAqbTl9KHYJzMySdg5Eza08SduJntk28/wN9NecnNO34sT3r+gtLhntNkZb9qTxFfdPqpkDaqWdb1voFhe/jKyaePabPk99AqKNp/n+GbBmzRoF0Zdeeqlx58aCaGCAZBLhNJeRRDYEgmiVJtaiWgyH8/j1FW2XcuVpC0QbT/P9M2D9+vUKojt37mzcOSBaVrKIj9KdWdM0H1BM4ntgdh7PGL1Z8CR1s4+5t7ruAwQg2nia758B1157rYLo9u3bGyDJVtMAABjISURBVHcOiJYBUQBaRmLOLLGqZdaFSC/VltEGdZQR7+PVAUQbT/P9M+Cmm25SEH3yyScbdw6IjpcQSMLoRQyMFwNAtPE03z8DNmzYoCA6Pz/fuHNAdLyEQAJFL2JgvBgAoo2n+f4ZcOeddyqIbt68uXHngOh4CYEEil7EwHgxAEQbT/P9M2DTpk0KonfddVfjzgHR8RICCRS9iIHxYgCINp7m+2fAww8/rCA6NzfXuHNAdLyEQAJFL2JgvBgAoo2n+f4ZsG3bNgXRG264oXHngOh4CYEEil7EwHgxAEQbT/P9M+CFF15QEL3iiisadw6IjpcQSKDoRQyMFwNAtPE03z8Ddu3apSAqTy5q+g+IjpcQSKDoRQyMFwNAtOks38P29+/fryB6wgknNO6dhqgEOv9oQAwQA1XEgOSZOv6+UUcjtNEOBY477jgF0qZ/U1SCm380KBoDd/+v/xtdcvYt0f+++f8RN4ydsWKgjswLROtQuSVtXHLJJQqii4uLLbEIM1BgtAK/+tWvVNzed999owtTAgVqVgCI1ix4k83dcsstKhndf//9TZpB2ygwlgJAdCy5KFyzAkC0ZsGbbO7xxx9XEL3uuuuaNIO2UWAsBYDoWHJRuGYFgGjNgjfZ3GuvvaYgeu655zZpBm2jwFgK/PKXv1Rx+8ADD4y1H4VRoA4FgGgdKrekjc8//1wlo8FgEH399dctsQozUGC4Aj/72c9U3G7dunV4Qb5FgQYUAKINiN5kk6effrpKSG+++WaTZtA2ChRW4JprrlEx+/zzzxfeh4IoUJcCQLQupVvSztVXX60S0q9//euWWIQZKDBcgUsvvVTF7J///OfhBfkWBRpQAIg2IHqTTT7xxBMqIf3whz9s0gzaRoHCCnzve99TMfv2228X3oeCKFCXAkC0LqVb0s67776rEtLhhx8effXVVy2xCjNQIF+Bk08+WcXsgQP1PIEm3xK+QYGsAkA0q0nvt5x99tkqKf3pT3/qva842G0FvvjiCxWrxxxzTLcdwfreKgBEe9u1+Y5t2LBBJaY2/EB3vpV8gwJRtHPnThWr69evRw4UaKUCQLSV3VKtUdu3b1eJafXq1dU2RO0osEwFfvGLX6hY5SlbyxSS3StTAIhWJm17K/7oo49UYpL7Rf/617+211Asm3oFVq1apWL11VdfnXotEKCdCgDRdvZL5VZddNFFKjlt2rSp8rZoAAUmUUAufDvssMMiuQjun//85yRVsA8KVK4AEK1c4nY2oJ+ju3LlyujLL79sp5FYNdUKPP300+pA7+KLL55qHXC+3QoA0Xb3T2XWyWP/zjvvPJWkHnvsscraoWIUmFSBCy64QMXn/Pz8pFWwHwpUrgAQrVzi9jbw0EMPqSQl5534Q4E2KfCXv/xFxabc2nLo0KE2mYYtKOAoAEQdOabrgzyQ/qyzzlLJSq7Y5Q8F2qLAtddeq+JSrs7lDwXarAAQbXPv1GCb3DogV+lefvnlNbRGEygwWgF5MpFcUCT/f/vb30bvQAkUaFABINqg+G1oWhLWKaecokD61FNPtcEkbJhyBfSPJPDj8VMeCB1xH4h2pKOqNPORRx5RED3ttNOi999/v8qmqBsFhiqwsLCgYvHoo4+OPv7446Fl+RIF2qAAEG1DL7TABjnql2Xd66+/vgXWYMI0KvDee+9FRx55pIpDfoB7GiOgmz4D0W72W+lWywxUZqICUrmHlD8UqFMBueVK/+TZD37wgzqbpi0UWJYCQHRZ8vVr523btimInnTSSdFbb73VL+fwptUK3HjjjSr2jj/++IifPGt1V2GcpwAQ9QSZ9o8333yzSmZycQd/KFCHArfffruKOVkFefHFF+tokjZQoDQFgGhpUvajok8++cQ8yeiee+7ph1N40VoFbIDyZKLWdhOGDVEAiA4RZ1q/ktmAzArk/84775xWGfC7QgX+/e9/RzfddJOJs7vvvrvC1qgaBapTAIhWp22na5Z7RjVIZbbAHwqUpcC//vWv6Ec/+pGJLwBalrLU04QCQLQJ1TvSpiyvaZDOzc11xGrMbLMCn332WbR27VoTV48++mibzcU2FBipABAdKdF0F5DfGwWk0x0DZXn/7rvvRmeccYaKpyOOOCJ67rnnyqqaelCgMQWAaGPSd6fhe++914D0jjvu6I7hWNoaBX7/+99H8hQiOSA78cQTozfffLM1tmEICixHASC6HPWmaF8bpPKePxQoooA8RGHDhg3mIOzcc8/lcX5FhKNMZxQAop3pquYNtUF6zTXXNG8QFrRagf3790cXXnihAaj8vNnBgwdbbTPGocC4CgDRcRWb8vK/+93vTFI8++yzebrMlMdDnvvPPPNMJE8fkuXbFStWRDwLN08ptnddASDa9R5swP6PPvpIndfSFxy98sorDVhBk21UQK6+lVUKHRvf/e53o3feeaeNpmITCpSiABAtRcbprOQnP/mJSZY8tH46Y8D2+o9//GP07W9/W8XEN7/5zejnP/95JOdE+UOBPisARPvcuzX49vDDDxuQ3nrrrdE//vGPGlqliTYpILPPG264wcSBgHTXrl1tMhFbUKAyBYBoZdJOT8V/+MMfzPKu3Ei/Z8+e6XF+yj21Z5+yhCu/S/v3v/99ylXB/WlSAIhOU29X6Ovbb78dXXbZZWo2cuyxx0YPPvjgyNaOOuqokWUo0E4F5OfK9A+5Czxl9rm4uNhOY7EKBSpUAIhWKO60VX3o0KHI/lWOmZmZaNhFR4cddlh00UUXTZtMrfZX//iAPFEo9CcPjv/Nb35jrrwVgMpS7ueffx4qzjYU6L0CQLT3XVy/gzIjWb16tTlHJg8YF8Daf5KkJQF/61vfih566CH7K943pICc29QXBknf+H9yla0c9Mh38n/aaadFL730kl+MzygwVQoA0anq7vqc/eqrryKBp064q1atinbu3KkM0Nv0qyzrfvjhh/UZR0tBBW677TZ1T6ful8MPP1yV030pBzzynWyXn8iT7fyhwLQrAESnPQIq9v/ll18250olAduPgNPJWmal69evr9gSqh+mgCy76/6wXwWcZ555pvlODobk/Dd/KIACsQJAlEioRYHNmzdHRx55pEnGdqKW9wJSOR/HX/0KyDLuySefnNs30j+yzLuwsBDJOVH+UAAFUgWAaKoF7ypW4I033hiaqOVXPiSh81evArKMO+wAR5Zvv/jii3qNojUU6IgCQLQjHdUHM+UZqv4M1P4s319xxRV9cLUzPuzdu3don+j+6YxDGIoCNSsARGsWfJqb0wl51CvLuvVFyTnnnFMIonm3vNRnKS2hQDsVAKLt7JfeWTUKnPb38qPNLOtWHwL33HOPusXI1n7U++qtogUU6JYCvYXormcPRPy3Q4Pvn3NLtPa8n6v/Nd/dEH3/nLlItuX+nzsXbfyf2+i/CmP4dw/9V77+Vt9ceu6GaM33bo3WnndbJO8ZU+0YU9IP/LVDgV5D9MWtByL+0YAYIAb6FgN3/ce+dhAEK6LeQ3RpKYr4RwNigBjoUwwA0fbQG4gCWQ4yiAFioGMxAESBaOUKyDkDWcLp09EnvjCbIgaIAYkBIFo5Qgo3wEy0Y0egJFGSKDFADADRwoyrvCAQBaLM1okBYqBjMQBEK2dj4QaAaMcGD7MQZiHEADEARAszrvKCQBSIMgshBoiBjsUAEK2cjYUbAKIdGzzMQpiFEAPEABAtzLjKCwJRIMoshBggBjoWA0C0cjYWbgCIdmzwMAthFkIMEANAtDDjKi8IRIFob2Yh++bXZX+RZO1CtK+0Pt4dzQ4G0cz8UoOaxTaYB8XP7W7QFmDW1AENEK2cjYUbAKKlJVgSSlMJRbcbQ3RdtGWP7oulaMvaQTQY2Nv0d5O8VgPR2O6N0eLIWBR/bF8SoALSqTuQAKKFGVd5QSA6MnFNkmzZR4OtztcsRKNoac9CNNP47HF4PBSHaLaexTk5SCgC4Oy+dfYNbZWrPxCtnI2FGwCiQLQ3R/FBiC6lszUNqy3Wsq9Zmk1ga5ZJvWXgeF8B1iAaeDM/57vA9wKQGHbJ/hbUM/sOBtHsjuIJV9Xr2QqwiuvXVa2AaGHGVV4QiALRfkPUmokaYPnQ2bFRwdEAdUkvA/szvGS7BdHsTDCBtmkjVJeUSZdlY7v8tgqAwPKtqzDA7gL9HMhRQLRyNhZuAIgGApSBPdnAblq3GEYpnJY8GGa/Fz8TyBnoJb4ngHJnhR5EVRm7vWRfBeVku/0+J9bGh6gGc6DtnDaa7hvaL3dMAdHCjKu8IBAl6fRsJpoumaqlVwuOYVjFM8d0FqqTXWi7C9G4Pq89Wc5V/zHgsjNVXX/6GrYr/T4fQP6st8g+lMnXszvaANHK2Vi4ASAKRHsG0fzZWRhWIVhKMg1tD0E0vz1J1kXOWYbtKpjQk6Vod8ZccF9iv7OxD0QLM67ygkCURNLZROLPKGIY5UMtDKsEjNaMVdVbZDm3AMBG2SRthe0qCMICNvg68bmgti3ODUC0cjYWbgCItnigkOzGS3ajgJULqwyIcsCqz59mLixywb04Z39Ollyd21CkfutCokz7OX4rsFv76ZmuU3fOvsR5bw4WJS8A0cKMq7wgECW59Ca5TAxRiYEEZHm3uMQHNO5yrj7Iic976nOhg2gggDQPfBCoJfuZ86Xy1KPd0T6rjFOHPyt2YlRDOWlvaFmAqvuob69AtHI2Fm4AiDoJiqTTt2RTrj9hiJbbBjGInqNjAIgWZlzlBYEoEO3NTLTy5Ft02ZWYIqYqjgEgWjkbCzcARCsO9soTO/ZXnrDjZWJZPrXPdY6eLdD3aFRVDADRwoyrvCAQBUKVQ6iqREK9QGpaYwCIVs7Gwg0AUSAKRIkBYqBjMQBECzOu8oJAtGODZ1qPvPGbWScxkMYAEK2cjYUbAKJAlFkIMUAMdCwGgGhhxlVeEIi2cPDkPhSgRFuLPI6OI//0yL8eLSq4BzT45KW6/cq2N5Xxp67uLufiNCBaORsLNwBEEzAVudld4GYeVC7Jacwb3dOrPK0b860b8OVGf3kGajGIJgnXenrOOIl+dBLzEnpiZ7ue0ZqngfdEIH3wYSexSW5XSfYxD2SwHjQfeqCCW0763H3akNtfWV+kj5atNxBtzyzbjj8dkxO+AtHCjKu8IBBNkoyBowS1BHsGkJKY06NIB6gTDYT8G/eLQTR/fzc5Z2cB8n1hiNqQ1hCxt5XsexHb7TLxwY8HpxxwOD7nlLHrzrxP/HfAFtommoxbv6onja9M2xPpPIEdk7Yz5n5OX4y5b2na1N1uiX0MRCtnY+EGphyi8dG/A9DcgbU7mrXA6j4fNQyq4YM9H4LFIDpJm+k+o5NYdmYk/gShlatZ2p6rRb7vbrm8/a3tgcQU6zeIBg7sx+lrq37btyAwwzqVA9G4bgfatj1F3o8L8yJ1llBmdPzl9EEJbY8dY2W1GYjVSW0BooUZV3nBqYZoMVglCd9bdtVLdcUAHEoI+SAxdiVJO27LnqUkiVtsMqDQkFiIZo2t1gzNq2tmrTzjdSHal5sgwnCIbUtsSRK01sJfrtR+bJlfl/zGpjwzdl+0Rdo2NqbLnOHEmq9TnIC030vJsl1cfmattJn1XwHJ0sIASvkiflnaDsReXW/6fF2zj9LObz/p67HhFdB7nDosn5S2um9NHUP8MgdHab84futn/84tWH1nxWPSRtqnlu46vhz71kWj4y/V29SrfbJ1n9sYzdixZMaD9EPSN0PLhMZmvC0+YByhiWnb0kP7nLSv7Y9j0itXRDtTX2orEK2cjYUbmGKIfhknBGdgpkHqHyHKgDLJUwLfGaz5+/n1pJ/z4RDDxwZk6Dypn3TTJGnsTAafri/dnrQ91He/fjuxSJKUOuyEkK1Tt5uFdY7vKtHaderlSG+bl1Rc+IrdUj62X/usgb6o902Suv5ezxxzwSv7+fsY+ATsM/AaIzYyCTVN4JKIja3aB/Ma++qCL2nX1GmBzbdN/LLjOeNn0l/+QUVBTXQcpPZnYyUdF1ov8cmyWQPJ2Bn77MaWv83/LHWHtuk29av2129f93O2joyPWndjrx7Duo7h8fT0W9qW8CsQLcy4ygsC0aEg0QEsgyodUDJg0oSgy4z7mgxUa5DpRJJJ+CZZWQNQJwOzfzKwzWdtT7zdT7AueHRZ+zVQX5Jc/bq03TFkUhtjP9LPppyZ2ez2LvqINbHrV3WM6iNlV9KOvE/KKx+VHgGtfVD4YFGA8rRL9tEzC/WaZ1uwPltf/73XlrRfuI5AXyn78+oItKXLq1dfr+Rzxtdsf8V9bNdvv099Hh1/aVkdN+64CNfrxmCRMtl23Dqy34fj2tUs6J8dp8kYsGPd1m72d596Y8O1A4hWzsbCDQDRTGJwg1UFtiQzU04GSwgMgf2cxOR/7w46nSjk1U0WyX5+0s+BaGZQOgM3tSE4yB174wTkACMzG0p8MEtaMnNKtQn6EUzSqV1qH0drb0nVsVHvF9sqBzbil9FAQUh+lmxBLfk5Bz6+nqEyvsbOPtr39ODK7sPiAIx9CCbmoE3aZ+81sc2ZSYtWwTqSvrUPuMz+1uzXfJ8Tq0ndfozoz6ofVL1pTGiNRsef+BeKQa13/J3pax0Xjr9Fyng6mtUF3U7O9yZG0+9Tn3LatbUYoR0QLcywxgtOMUSLXSSjBoYDiTTJOElZD+LCrzmJqW0QNYk0TRZxIkwSnJ1M7CSR54fSJ993J+mrRJOfzHRCllfVT3Mbk6VcbWvcTub8qNiQQMP0oZN89f6Jj1qDnH2cpVDd/8H6dL3Z1+ABx5h1iA5xPdapgGAdrl96H6NFZqUgp7+SujMg0xoYnSeAaFK3ra2rUQ6oHH+LlMn2RQrD7Hcm1uy4T/xN98tp1x4fRbSzdfTeMxNtnJ3GgKmGqE6kQ5NAkqBNgtmxMZ3peIFtJ/XR73MSk0mEHjz8BK6P0nWCTz5nfHGSSpoU0gGfbnNtdhOt+52GkJscVZ3LnIlKO6qeud0xEIx/eXYm2xN99FKutlcDwk7G6jtfz6BOngb+PqavXB1U/cH68n2I7fTqGbMO7bOOaxWzwTpsvwJxmOyTahYoo2LfrifHt2D7SR8HQKR9cIEpdSc2mPOkcdt+vLs6FimTtduto+j3tkb2e2t/FT+6jwtoNyS/AFHDsMbfTDdEdcL2L5hQA19DTIJdv48HvwGqFeQ+QHQyyH/NGWgmMadtqjoyCdwfhOGEkSYfPXithDQkiZmltDyI+clRDi6SqyC1PtlEmCaUWC/PR61nkmzkCk5dV76Ous4c/307nTas+oPlPI0zfSBtJ2V8LYP1aVtDr15bys7QttC+3jbbzqAdbr2qL4z9Epcbo9k5++rt/FjVwHZg5hxoJvtaB1cmJk2bnv3iexIDW/bE30kszc5ttE4XuD6o+Eh8TeFfpEygbd2n1riPbdbxmq3XB2/82Tq1YMa1NQ6TfsrXLmRbvA2INs5OY8DUQ9QGlD6XI68z8wvR4o5kMBuQyODRA8kN8BgK7qAZnvjzE1MQPnZiDCbYeGA7A1IDwzpYiH0scotBNlH4/uhEoepMNLITctAPY1NSf7JU7tqdfDcsyZp60n4I37sbQ8Fclav38/UsABsNjAzYk7rS5J13LjK11ddSfdYQMKcPJNYsnUwc+vVYZXw9i/hlt6Fh5+yXH6vKbu2/sVsuvLNuDZoo/pLZquOPbUfsc7xUn55icfog8Wt4GV9L/Tlpy/gkY3t3tC+Bujl4Mt9bcPRiTOeV4C0uBbQLxQoQNQxr/A0Q1QG/7Nd40GUS7LLr1YN6ml5jLV2wTpP/+BoCh7stgah9H29mrBUp002tgWjj7DQGANHMwJtwUMkR5ZgzJzcpTNhuWfa3qR51dB44sm+Tjdgy9BaM6mO7CCCLlOnmuAOihmGNvwGiZSRDAFpSQo2TXuY2jTL6iDpK6qO2QKcIIIuUaYs/49kBRBtnpzEAiJJce5Zcx0tG1c+YsAeNy48BIGoY1vgbIApEgSgxQAx0LAaAaOPsNAYA0Y4NHo7qyz+qR1M07VoMAFHDsMbfAFEgyiyEGCAGOhYDQLRxdhoDgGjHBk/Xjpixl1keMVB+DABRw7DG3wBRIMoshBggBjoWA0C0cXYaA4BoxwYPR/XlH9WjKZp2LQaAqGFY42+AKBBlFkIMEAMdiwEg2jg7jQG9h+iLWw9E/KMBMUAM9CkGgKhhWONveg3RXc8eiPhHA2KAGOhjDDRODwxQCvQWovQvCqAACqAAClStABCtWmHqRwEUQAEU6K0CQLS3XYtjKIACKIACVSsARKtWmPpRAAVQAAV6qwAQ7W3X4hgKoAAKoEDVCgDRqhWmfhRAARRAgd4qAER727U4hgIogAIoULUCQLRqhakfBVAABVCgtwoA0d52LY6hAAqgAApUrQAQrVph6kcBFEABFOitAkC0t12LYyiAAiiAAlUrAESrVpj6UQAFUAAFeqvA/we+SPO1cAhxigAAAABJRU5ErkJggg==">

Reply:
We'll need to change the third box to not say 'C#'. It can be a c#/winrt or c++/winrt or any kind of project as long as it produces winrt.

Reply:
Oh right I used mermaid diagrams in github to generate the image:

```mermaid
flowchart TD
 A[Project 1\nC#, App Code] --> B[Project 2\nC++/winrt, wv2winrt generated code]
 A --> C[Project 3\nThird Party WinRT & shared app code]
 B --> C
```

Reply:
handled in my png commit

Reply:
switching to bottom png...